server/project-service/src/main/java/edu/illinois/ncsa/incore/service/project/models/WorkflowResource.java

```diff
@@ -53,6 +53,10 @@ public String getTitle() {
         return title;
     }
 
+    public boolean getIsFinalized() {
```

Comment:
I made all the fields public so technically you don't need to have a getter/setter method. You should be able to directly say `workflow.isFinalized`. But it doesn't hurt.